components/hook/usePortal/src/index.js

```diff
@@ -202,7 +202,7 @@ const usePortal = ({
             >
               {children}
             </As>,
-            target
+            target || document.body
```

Comment:
IMO this line will never be executed on server cuz `isReady`  L194 condition prevents that

Reply:
I'm not sure on `Portals`, but we use same approach when we have some `<Suspense>` on SSR's pages. So, let's go! 🚀